### ISSUE TITLE
Feature Req: Change coverage badge text

### ISSUE THREAD

**Issue by DevExperte** [cicirello/jacoco-badge-generator#41]
We have multiple workfows in a single repository creating different coverages. We'd like to set the label text on the badge to show which coverage was calculated. E.g. instead of
> coverage: 70,4%

we'd like to change the text to

> coverage (product.core): 70,4%

I found no possible solutions to this problem in your documentation, maybe there is none yet!

**Comment by cicirello** [comment]
At the present time, there isn't an option to change the label text. If you want a workaround for now, you can configure it to generate [JSON instead of directly generating the badge](https://github.com/cicirello/jacoco-badge-generator#direct-badge-generation-vs-json-endpoint) by setting `generate-coverage-badge: false` and setting `generate-coverage-endpoint: true`. You would then pass the URL to the JSON file to Shield's endpoint, which allows overriding the label.

For example, consider the badge JSON file, https://github.com/cicirello/jacoco-badge-generator/blob/main/tests/999.json, with raw URL, https://raw.githubusercontent.com/cicirello/jacoco-badge-generator/main/tests/999.json.

You pass alone to Shields, with this markdown:
```Markdown
![](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcicirello%2Fjacoco-badge-generator%2Fmain%2Ftests%2F999.json)
```
And you'll get: ![](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcicirello%2Fjacoco-badge-generator%2Fmain%2Ftests%2F999.json). 

But you can override the label with Shields's query parameter with this Markdown:
```Markdown
![](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcicirello%2Fjacoco-badge-generator%2Fmain%2Ftests%2F999.json&label=coverage%20%28product.core%29)
```
And you'll instead get: ![](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcicirello%2Fjacoco-badge-generator%2Fmain%2Ftests%2F999.json&label=coverage%20%28product.core%29)

**Comment by cicirello** [comment]
@DevExperte @morapet custom labels for left side of badges is now available in version 2.6.0. 🚀  New inputs `coverage-label` and `branches-label` control the new feature. Defaults are as before.
